src/main/frontend/components/settings.cljs

```diff
@@ -27,29 +27,29 @@
             [rum.core :as rum]
             [frontend.mobile.util :as mobile-util]))
 
-(rum/defcs set-email < (rum/local "" ::email)
-  [state]
-  (let [email (get state ::email)]
-    [:div.p-8.flex.items-center.justify-center
-     [:div.w-full.mx-auto
-      [:div
-       [:div
-        [:h1.title.mb-1
-         "Your email address:"]
-        [:div.mt-2.mb-4.relative.rounded-md.max-w-xs
-         [:input#.form-input.is-small
-          {:autoFocus true
-           :on-change (fn [e]
-                        (reset! email (util/evalue e)))}]]]]
-      (ui/button
-        "Submit"
-        :on-click
-        (fn []
-          (user-handler/set-email! @email)))
-
-      [:hr]
-
-      [:span.pl-1.opacity-70 "Git commit requires the email address."]]]))
+;; (rum/defcs set-email < (rum/local "" ::email)
```

Comment:
Should we remove commented out code in this ns and their associated translations? If not, a comment on why we still have this code would be helpful

Reply:
updated